golang/cosmos/x/swingset/types/query.pb.gw.go

```diff
@@ -534,15 +551,15 @@ func RegisterQueryHandlerClient(ctx context.Context, mux *runtime.ServeMux, clie
 }
 
 var (
-	pattern_Query_Params_0 = runtime.MustPattern(runtime.NewPattern(1, []int{2, 0, 2, 1, 2, 2}, []string{"agoric", "swingset", "params"}, "", runtime.AssumeColonVerbOpt(false)))
+	pattern_Query_Params_0 = runtime.MustPattern(runtime.NewPattern(1, []int{2, 0, 2, 1, 2, 2}, []string{"agoric", "swingset", "params"}, "", runtime.AssumeColonVerbOpt(true)))
 
-	pattern_Query_Egress_0 = runtime.MustPattern(runtime.NewPattern(1, []int{2, 0, 2, 1, 2, 2, 1, 0, 4, 1, 5, 3}, []string{"agoric", "swingset", "egress", "peer"}, "", runtime.AssumeColonVerbOpt(false)))
+	pattern_Query_Egress_0 = runtime.MustPattern(runtime.NewPattern(1, []int{2, 0, 2, 1, 2, 2, 1, 0, 4, 1, 5, 3}, []string{"agoric", "swingset", "egress", "peer"}, "", runtime.AssumeColonVerbOpt(true)))
 
-	pattern_Query_Mailbox_0 = runtime.MustPattern(runtime.NewPattern(1, []int{2, 0, 2, 1, 2, 2, 1, 0, 4, 1, 5, 3}, []string{"agoric", "swingset", "mailbox", "peer"}, "", runtime.AssumeColonVerbOpt(false)))
+	pattern_Query_Mailbox_0 = runtime.MustPattern(runtime.NewPattern(1, []int{2, 0, 2, 1, 2, 2, 1, 0, 4, 1, 5, 3}, []string{"agoric", "swingset", "mailbox", "peer"}, "", runtime.AssumeColonVerbOpt(true)))
 
-	pattern_Query_Storage_0 = runtime.MustPattern(runtime.NewPattern(1, []int{2, 0, 2, 1, 2, 2, 2, 3, 1, 0, 4, 1, 5, 4}, []string{"agoric", "swingset", "storage", "data", "path"}, "", runtime.AssumeColonVerbOpt(false)))
+	pattern_Query_Storage_0 = runtime.MustPattern(runtime.NewPattern(1, []int{2, 0, 2, 1, 2, 2, 2, 3, 1, 0, 4, 1, 5, 4}, []string{"agoric", "swingset", "storage", "data", "path"}, "", runtime.AssumeColonVerbOpt(true)))
 
-	pattern_Query_Keys_0 = runtime.MustPattern(runtime.NewPattern(1, []int{2, 0, 2, 1, 2, 2, 2, 3, 1, 0, 4, 1, 5, 4}, []string{"agoric", "swingset", "storage", "keys", "path"}, "", runtime.AssumeColonVerbOpt(false)))
+	pattern_Query_Keys_0 = runtime.MustPattern(runtime.NewPattern(1, []int{2, 0, 2, 1, 2, 2, 2, 3, 1, 0, 4, 1, 5, 4}, []string{"agoric", "swingset", "storage", "keys", "path"}, "", runtime.AssumeColonVerbOpt(true)))
```

Comment:
I would not expect these lines to change.

Reply:
And now they don't, given the fix above.